pkg/deployd/deployd/deployd.go

```diff
@@ -53,13 +53,15 @@ func jsonToResources(json []json.RawMessage) ([]unstructured.Unstructured, error
 }
 
 // Annotate a resource with the deployment correlation ID.
-func addCorrelationID(resource *unstructured.Unstructured, correlationID string) {
-	anno := resource.GetAnnotations()
-	if anno == nil {
-		anno = make(map[string]string)
+func AddAnnotations(resource *unstructured.Unstructured, annotations *map[string]string) {
```

Comment:
- This function does not need to be public/exported (capital first letter in Go makes a function public)
- Maps are always passed by reference, no pointer needed

Reply:
Nice! Nyttig å vite! =)

Reply:
Done!